package.json

```diff
@@ -4,18 +4,20 @@
   "version": "0.2.0",
   "license": "Apache-2.0",
   "dependencies": {
-    "@google-cloud/datastore": "^0.8.0",
-    "body-parser": "^1.4.3",
-    "express": "^4.5.1",
-    "todomvc": "^0.1.0",
+    "@google-cloud/datastore": "^6.3.0",
+    "body-parser": "^1.19.0",
+    "express": "^4.17.1",
+    "todomvc": "^0.1.1",
     "todomvc-api": "^0.2.1"
   },
   "devDependencies": {
-    "gulp": "^3.8.8",
-    "gulp-jshint": "^1.8.5"
+    "mocha": "^8.2.1",
+    "standard": "^16.0.1"
   },
   "scripts": {
-    "start": "gulp serve",
-    "test": "gulp test"
+    "start": "node index.js",
+    "test": "mocha",
+    "lint": "standard",
```

Comment:
nit: what about https://www.npmjs.com/package/semistandard ? (or simply eslint)

So that it matches the rest of the node.js samples for cloud run:
https://github.com/GoogleCloudPlatform/nodejs-docs-samples/blob/master/run/helloworld/package.json

Reply:
Yeah, I was trying to be as lazy as possible and go with the no config, one package solution 😆 I'm with you on the semistandard thing, that was annoying.